cnf-tests/testsuites/e2esuite/dpdk/dpdk.go

```diff
@@ -272,6 +272,14 @@ var _ = Describe("dpdk", func() {
 				// In case of nodeselector set, this pod will end up in a compliant node because the selection
 				// logic is applied to the workload pod.
 				pod := pods.DefineWithHugePages(namespaces.DpdkTest, dpdkWorkloadPod.Spec.NodeName)
+
+				// The pod needs to request the same sriov device as the dpdk workload pod
+				// using the topology manager it will ensure the hugepages allocated to this pod
+				// are in the same numa.
+				pod.Annotations = map[string]string{
```

Comment:
You can use pods.RedefinePodWithNetwork here

Reply:
done can you please have another look?